ADRs/ADR0006-replace-badger-kv-store.md

```diff
@@ -0,0 +1,144 @@
+# Replace Badger KV Store with PostgreSQL
```

Comment:
I feel like the title and filename should be a condensation of the summary down to as few but **as meaningful** words as possible.

I'm trying to imagine reading the filename 6 months from now. I'm fairly confident that I'm going to have to open the document before I can recall anything useful about the decision. When I imagine the same scenario but having instead used a name like `use-postgres-for-smt` or `use-postgres-kv-store`, I feel like there's more context present sooner, if that makes sense. I think we should try to imagine how each filename would read in the context of a large directory full of ADRs. (Out of scope for now but perhaps we should consider some form of organization sooner rather than later.)

Wdyt about phrasing it in terms of the direction we're going towards rather than away from?

Reply:
That's an interesting distinction between coming from and going to. I would be fine with the style of forward-looking titles. I went with `Replace` because in this context the main issue is that it's not just a new implementation but a complete engine swap of the KV store. If we think forward-looking is better, I can change it for sure.

Reply:
Perfect :sunglasses::ok_hand: